psd-web/app/helpers/investigations/display_text_helper.rb

```diff
@@ -3,6 +3,26 @@ def image_document_text(document)
     document.image? ? "image" : "document"
   end
 
+  def investigation_sub_nav(investigation)
+    items = [{ href: investigation_url(investigation), text: "Overview", active: @current == @investigation.pretty_id },
+             { href: investigation_products_url(investigation), text: "Products", count: " (#{@investigation.products.count})", active: @current == "products" },
+             { href: investigation_businesses_url(investigation), text: "Businesses", count: " (#{@investigation.businesses.count})", active: @current == "businesses" },
+             { href: investigation_attachments_path(investigation), text: "Attachments", count: " (#{@investigation.documents.count})", active: @current == "attachments" },
+             { href: "/cases/#{@investigation.pretty_id}/activity", text: "Activity", active: @current == "activity" }].compact
+    render "components/hmcts_sub_nav", title: "Summary", items: items
+  end
+
+  def investigation_sub_nav_tabs
+    mapping = {
+      products: :products,
+      businesses: :businesses,
+      attachments: :attachments,
+      activity: :activity
+    }
+
+    render "investigations/tabs/" + mapping.fetch(@current.to_sym, :overview).to_s
```

Comment:
```suggestion
    render "investigations/tabs/" + @current || 'overview')
```
and remove the `mapping`